gui/wxpython/image2target/ii2t_manager.py

```diff
@@ -153,13 +154,13 @@ def __init__(self, parent, giface):
         self.target_gisrc = os.environ["GISRC"]
         self.gisrc_dict = {}
         try:
-            f = open(self.target_gisrc)
-            for line in f:
-                line = line.replace("\n", "").strip()
-                if len(line) < 1:
-                    continue
-                key, value = line.split(":", 1)
-                self.gisrc_dict[key.strip()] = value.strip()
+            with open(self.target_gisrc) as f:
+                for line in f:
+                    line = line.replace("\n", "").strip()
+                    if len(line) < 1:
+                        continue
+                    key, value = line.split(":", 1)
+                    self.gisrc_dict[key.strip()] = value.strip()
         finally:
             f.close()
```

Comment:
What should happen if the try fails? Is the rest of the function / callers working correctly if it silently fails?